nights/2020-01-05-README.md

```diff
@@ -1,34 +1,71 @@
 # 2020-01-05
 
-## Reviewed by:   your_name_here
+## Reviewed by:   Max Thompson
 
 ## Raw image gallery: http://physics.mnstate.edu/feder_gallery/2020-01-05
 
 ## Unusual images?
-
 Are there any images that look unusual? List the file name of any unusual images for this night here, with description:
+The first 11 Bias images are normal FYI other than #5
++ Bias-S001-R001-C005-NoFilt.fit - Darker horizontal band in lower 3rd
```

Comment:
Can you please add backticks around each image name, as shown below? That makes the names easier to select later on.

```suggestion
+ `Bias-S001-R001-C005-NoFilt.fit` - Darker horizontal band in lower 3rd
```